core/object/message_queue.cpp

```diff
@@ -125,7 +111,7 @@ Error MessageQueue::push_set(Object *p_object, const StringName &p_prop, const V
 	return push_set(p_object->get_instance_id(), p_prop, p_value);
 }
 
-Error MessageQueue::push_callable(const Callable &p_callable, const Variant **p_args, int p_argcount, bool p_show_error) {
+Error MessageQueue::push_callablep(const Callable &p_callable, const Variant **p_args, int p_argcount, bool p_show_error) {
```

Comment:
For now it's fine for compatibility, but eventually we could likely remove the args pointer and just make it `push_callable(const Callable &p_callable, bool p_show_error)`. We can pass args via `Callable::bind`.

Reply:
Yeah I am honestly not sure how it is supposed to be used, although the unlimited varargs now should resolve most of the original need for it.